utils/nctl/sh/scenarios/slow-node-01.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+source "$NCTL"/sh/scenarios/common/itst.sh
+
+# Exit if any of the commands fail.
+set -e
+
+#######################################
+# Runs an integration tests that tries to simulate
+# a slow validator in the network and check if it did
+# slow down.
+# Arguments:
+#   `timeout=XXX` timeout (in seconds) when syncing.
+#######################################
+function main() {
+    log "------------------------------------------------------------"
+    log "Starting Scenario: slow-node-01"
+    log "------------------------------------------------------------"
+
+    # 0. Wait for network start up
+    do_await_genesis_era_to_complete
+    # 1. Verify network is creating blocks
+    do_await_n_blocks "5"
+    # 2. Verify network is in sync
+    parallel_check_network_sync 1 5
+    INIT_ROUND_LEN=$(get_node_round_len 1)
+    # 3. Slow down node
+    slow_down_node_network 1
+    # 4. Wait 2 eras
+    do_await_era_change 2
+    # Dump consensus data from node 2 for era 2 for debugging
+    dump_consensus 2 2
+    # 5. Verify all nodes are in sync
+    parallel_check_network_sync 1 5
+    # 6. Check that round length of node 1 is different
```

Comment:
Does it make sense to also check whether the round length on other nodes did not change?

Reply:
That's a good idea.
Although I think in this test the round lengths on other nodes change sometimes, because the slow node has 20% of the total weight. We'd have to change a few more things so that the expectation would actually be justified.